dp/drivers/ixgbe.c

```diff
@@ -1113,8 +1114,12 @@ int ixgbe_init(struct pci_dev *pci_dev, struct ix_rte_eth_dev **ethp)
 
 	if (!strcmp(driver->name, "rte_ixgbe_pmd"))
 		dev->dev_ops = &eth_dev_ops;
-	else if (!strcmp(driver->name, "rte_ixgbevf_pmd"))
+	else if (!strcmp(driver->name, "rte_ixgbevf_pmd")){
 		dev->dev_ops = &vf_eth_dev_ops;
+		/* check that the config is right */
+		if (CFG.num_cpus > 4)
+			log_warn("using more than 4 cores on a VF is currently not supported.\n");
```

Comment:
-  warning in case we wanted to add a check for this in the tests to track it instead of blocking it. I think right now we do not want this so I will update.
-  just tried to put twice the memory and it fails the same.
-  We can currently run only 1 IX with up to 4 cores. second ix craches because dpdk is not set up for multiprocess right now (we can update this but it require some meaningful code change in IX init path I think.)